modules/Bio/EnsEMBL/Production/Pipeline/Webdatafile/WebdataFileGenerateGC.pm

```diff
@@ -0,0 +1,142 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+ Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFileGenerateGC;
+
+=head1 DESCRIPTION
+  Compute Gene and Transcript  step for webdatafile dumps
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::Webdatafile::WebdataFileGenerateGC;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Registry;
+use base qw/Bio::EnsEMBL::Production::Pipeline::Common::Base/;
+use Path::Tiny qw(path);
+use Carp qw/croak/;
+use JSON qw/decode_json/;
+use Bio::EnsEMBL::Production::Pipeline::Webdatafile::lib::GenomeLookup;
+use Bio::EnsEMBL::Production::Pipeline::Webdatafile::lib::IndexWig;
+use Capture::Tiny qw/capture/;
+use Path::Tiny qw/path tempfile/;
+
+sub param_defaults {
```

Comment:
Not sure this overriden method is useful, since doing nothing more than parent?

Reply:
i will remove it